### PR TITLE
Use hostname es website title

### PR DESCRIPTION
**Pull request by marcohorstmann**
Because I have now two boxes (lottibox and marcobox) I need to see on which box I am. So best is to use hostname of the PI as title.

### CODE REVIEW

**Comment by marcohorstmann** [comment]
Why it showes me the old commits? I thought only c2b623d will be commited. What I'm doing wrong? @MiczFlor

File: scripts/daemon_rfid_reader.py [hunk 1]
```diff
@@ -9,11 +9,23 @@
 
 print dir_path
 
+
+base_path = os.path.dirname(dir_path)
+
+try:
```
Comment (MiczFlor) [inline scripts/daemon_rfid_reader.py]: @marcohorstmann 
isn't this always overwriting these files with an empty string each time a card is swiped?

**Comment by MiczFlor** [comment]
Another thing, you suggested somewhere sometime to change the systemd servies to the phoniebox namespace. I did. Please have a look here and tell me if this is what you had in mind:
https://github.com/MiczFlor/RPi-Jukebox-RFID/commit/55bc7492b030702d440ed99c423ceb24c032fca7

**Comment by marcohorstmann** [comment]
btw. @MiczFlor maybe you can explain me how I can contribute without generating any mess? I bought an extra Pi for myself and making development on my box so I have not to grab the box of my daughter every evening. ;-)

**Comment by MiczFlor** [comment]
Hi @marcohorstmann 
sorry for the silence, I was away over the weekend. Best thing to do would be to install the develop branch version, switch to the develop branch and keep changing and contributing there. If you have a dedicated dev hardware: awesome :)
How would you do that?
If you already have the system installed, try this:
~~~
cd /home/pi/RPi-Jukebox-RFID
git checkout develop
git fetch origin
git reset --hard origin/develop
git pull
~~~
If you have an empty SD card with only stretch on it, best to install from scratch with the one line install script from the develop branch
## without spotify:
~~~
cd; rm stretch-install-*; wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/develop/scripts/installscripts/stretch-install-default.sh; chmod +x stretch-install-default.sh; ./stretch-install-default.sh
~~~
## with spotify:
~~~
cd; rm stretch-install-*; wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/develop/scripts/installscripts/stretch-install-spotify.sh
~~~
And then go to the dev branch :
~~~
cd /home/pi/RPi-Jukebox-RFID
git checkout develop
git fetch origin
git reset --hard origin/develop
git pull
~~~
You can check the branch you are on with 
~~~
git branch
~~~
Hope this helps :)

**Comment by MiczFlor** [comment]
@marcohorstmann 
ich mach das hier jetzt mal zu, denn ich merge den ganzen Spotify Code aus `develop` auf `master`. Dein eÄnderungen müsstest du dann auf `develop` noch mal einbauen. Danke

**Pull request closed without merging**